GS_example.py

```diff
@@ -36,21 +40,27 @@
           np.arange(-params.matrix_size / 2, params.matrix_size / 2) * params.pixel] for y in
          np.arange(-params.matrix_size / 2, params.matrix_size / 2) * params.pixel])
 
+    # Prepare optimizer
     GS = GerchbergSaxton(params)
 
+    # Run optimizer
     res = GS.optimize(LightField(amp, phase), LightField(target, phase), 5)
 
+    # Plot the result - optimized phase map
     plotter = GeneratePropagationPlot(res[0], output_type=GeneratePropagationPlot.PLOT_PHASE)
```

Comment:
`res[0]` - maybe it would be nice to have destructured return value to make it more descriptive, so its rather a fix to line 47:
```python
in, out = GS.optimize(...)
```
Adjust the names accordingly - tbh I dont really know that this `GS.optimize` returns and I'm too lazy to check now :D

Reply:
Excellent idea

Reply:
Sure, I think  'input_plane, output_plane' will describe it most clearly.